streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/metrics/flink/CheckPoints.java

```diff
@@ -82,5 +96,20 @@ public String getPath() {
   @Data
   public static class Latest implements Serializable {
     private CheckPoint completed;
+    private CheckPoint savepoint;
+
+    @JsonIgnore
+    public List<CheckPoint> getLatestChkSavepointByIdOrder() {
```

Comment:
how about `getLatestCheckPoint` ?  The method name `getLatestChkSavepointByIdOrder`  is too long. As a method, it should not expose more internal implementation details. getLatest has expressed the purpose of the method, So I think it is don't necessary to add "ByIdOrder"

Reply:
Okay~.